### DISCUSSION TITLE
⚡️  Xcode 15.2 will be default starting February 4

### DISCUSSION THREAD

**Discussion by helinanever** [codemagic-ci-cd/codemagic-docs#2320] [Announcements]
As of February 4, 2024, we are introducing the Xcode 15.2 build machine image as our new default. Additionally, Xcode latest will be switched to `15.2`.

Key points:

* If your Xcode version is set to `latest` or if no specific Xcode version is configured, then your builds will utilize the Xcode 15.2 image.
* Refer to the [build machine specs](https://docs.codemagic.io/specs/versions-macos/) for details on preinstalled software and available runtimes.

Action required:

* Please review your build configuration promptly to avoid unexpected behavior.
* You can always set a specific Xcode version in your build settings. Configure this in the Build section of the Flutter workflow editor or the environment section of codemagic.yaml.

If you have any questions or concerns, don't hesitate to get in touch with us.

**Comment by mikhail-tokarev** [comment]
> Please note that as of April 2024 all iOS and iPadOS apps submitted to the App Store must be built with a minimum of Xcode 15 and the iOS 17 SDK.

https://developer.apple.com/ios/submit/#:~:text=Please%20note%20that,iOS%2017%20SDK
